### ISSUE TITLE
UKI: kairos-agent reset

### ISSUE THREAD

**Issue by mudler** [kairos-io/kairos#2051]
Follow-up of #1151 #872 

Depends on: #2226

There should be a `kairos-agent` CLI command that allows to set the next boot selection entry in order to replace instructions using GRUB https://kairos.io/docs/reference/reset/#remotely-via-command-line.

the reset shim has also to take into account of things like setting up the next boot entry hybrid with GRUB/systemd-boot

## Acceptance Criteria

- There is a `kairos-agent` command that replaces [this process](https://kairos.io/docs/reference/reset/#remotely-via-command-line) both for UKI and non-UKI systems.
- We have a separate binary that does implement the shim so it can be used for backward compatibility during upgrades cc @vipsharm

**Comment by jimmykarily** [comment]
We have "reset" but it doesn't work exactly like the "reset" in non-UKI installations. We need to discuss whether the current implementation is enough or if we need to do more work. We will plan a call for this one.

**Comment by jimmykarily** [comment]
Keeping open until it's bumped in kairos

**Comment by Itxaka** [comment]
this + shim is part of agent v2.7.13

**Comment by Itxaka** [comment]
v2.7.13 merged into packages, should shortly be available under framework

**Comment by Itxaka** [comment]
https://github.com/kairos-io/kairos/pull/2278

**Comment by Itxaka** [comment]
Now in master

**Comment by jimmykarily** [comment]
I did a test from master and it works. All good.
